### DISCUSSION TITLE
Unable to install Nora in my Garuda Arch Linux

### DISCUSSION THREAD

**Discussion by xandemon** [Sandakan/Nora#212] [Q&A] [answered]
I am using Garuda Arch Linux and I wanted to try Nora in my linux. Only AppImage was available for my system as deb doesn't work in arch, and I tried to use that but the AppImage didn't work. When I tried to open it, nothing happened. When I viewed the console, it gave some sharp module error. Can you prepare one for arch or add Nora in official aur repositories so I can directly install it using pacman?

**Comment by faveoled** [comment, marked as answer]
Try this:
```
sudo pacman -S libvips
```
